client/client_opts.go

```diff
@@ -243,6 +252,14 @@ func WithMaxConcurrentDownloads(max int) RemoteOpt {
 	}
 }
 
+// WithMaxConcurrentDownloads sets max concurrent download limit.
```

Comment:
Is this comment correct (as well as WithLimiter's)? It seems like a copy+paste from `WithMaxConcurrentDownloads`

Reply:
fixed, ty !